Messages/Messages.tsx

```diff
@@ -66,14 +67,10 @@ const cdHelp = `
 const commonCommands = `
   ${Prefix.PREFIX}${Commands.HELP} - this output
   ${Prefix.PREFIX}${Commands.JOIN} - connect wallet to the terminal
-  ${Prefix.PREFIX}${
-  Commands.BALANCE
-} gton | sgton | harvest | gcd | all - get actual erc20 token balance
-  ${Prefix.PREFIX}${Commands.ADD_TOKEN} gton | sgton | gcd | usdc  - add tokens to metamask,
-  ${Prefix.PREFIX}${
-  Commands.BUY
-} gton for <amount> <token>  - available tokens: eth, usdc (more coming soon) 
-  ${isTestnet ? `${Prefix.PREFIX}${Commands.SWITCH} - switch chain to GTON` : ''}
+  ${Prefix.PREFIX}${Commands.BALANCE} gton | sgton | harvest | gcd | all - get actual erc20 token balance
+  ${Prefix.PREFIX}${Commands.BUYANDBRIDGEGCD} <amount> <token> - buy GCD with stable at ~0.92 liquidation price and bridge it to rollup
+  ${Prefix.PREFIX}${Commands.ADD_TOKEN} gton | sgton | gcd | usdc - add tokens to metamask
+  ${Prefix.PREFIX}${Commands.BUY} gton for <amount> <token>  - available tokens: busd, usdc (more coming soon) 
```

Comment:
Do we have pool GTON/BUSD to buy for busd?

Reply:
No, just the stable protocol